crates/cext/src/transpiler/target.rs

```diff
@@ -444,6 +477,38 @@ pub extern "C" fn qk_target_entry_new(operation: StandardGate) -> *mut TargetEnt
     Box::into_raw(Box::new(TargetEntry::new(operation)))
 }
 
+/// @ingroup QkTargetEntry
+/// Creates a measurement entry for a ``QkTarget``.
+///
+/// @return A pointer to the new ``QkTargetEntry`` for a measurement instruction.
+///
+/// # Example
+///
+///     QkTargetEntry *entry = qk_target_entry_new_measure()();
+#[no_mangle]
+#[cfg(feature = "cbinding")]
+pub extern "C" fn qk_target_entry_new_measure() -> *mut TargetEntry {
+    Box::into_raw(Box::new(TargetEntry::new_instruction(
+        StandardInstruction::Measure,
+    )))
+}
+
+/// @ingroup QkTargetEntry
+/// Creates a reset entry for a ``QkTarget``.
+///
+/// @return A pointer to the new ``QkTargetEntry`` for a reset instruction.
+///
+/// # Example
+///
+///     QkTargetEntry *entry = qk_target_entry_new_reset();
```

Comment:
These examples aren't much of an example. Maybe show adding a couple qubits with an error rate and adding that to a target?

Reply:
Added in 0640bbc